src/redux/reducers/componentConfig/updateConfig.js

```diff
@@ -1,16 +1,21 @@
 import _ from 'lodash';
 
 const updateConfig = (state, action) => {
-  const { plotUuid, configChange } = action.payload;
+  const { plotUuid, configChange = null, dataChange = null } = action.payload;
```

Comment:
my understanding is that `configChange` is a boolean, is that the case? If that is the case, shouldn't the default value for it be either `true` or `false`?

Reply:
configChange is an object with all the changes in the plot's styling config, since now we need to be able to update the plotData too then we added dataChange.

Reply:
with risk of being a bit pedantic, if you rename it to `configChanges` and `dataChanges`, that might make it much clearer what these are straight-away